rfcs/0000-rfc-process.md

```diff
@@ -0,0 +1,127 @@
+- Feature Name: rfc-process
+- Start Date: 2017-02-12
+- Authors: zimbatm, ???
+- RFC PR:
+- Related Issue:
+
+# Summary
+[summary]: #summary
+
+The "RFC" (request for comments) process is intended to provide a consistent
+and controlled path for new features to enter the nix language, packages and
+OS, so that all stakeholders can be confident about the direction the
+ecosystem is evolving in.
+
+# Motivation
+[motivation]: #motivation
+
+There are a number of changes that are significant enough that they could
+benefit from wider community consensus before being implemented. Either
+because they introduce new concepts, big changes or are controversial enough
+that not everybody will agree on the direction to take.
+
+# Detailed design
+[design]: #detailed-design
+
+Many changes, including bug fixes and documentation improvements can be
+implemented and reviewed via the normal GitHub pull request workflow.
+
+Some changes though are "substantial", and we ask that these be put through a
+bit of a design process and produce a consensus among the Nix community and
+the core team.
+
+This is the bulk of the RFC. Explain the design in enough detail for somebody
+familiar with the ecosystem to understand, and implement.  This should get
+into specifics and corner-cases, and include examples of how the feature is
+used.
+
+## When you need to follow this process
+
+You need to follow this process if you intend to make "substantial" changes to
+the Nix ecosystem. What constitutes a "substantial" change is evolving based
+on community norms, but may include the following.
+
+  - Any semantic or syntactic change to the language that is not a bugfix.
+  - Removing language features
+  - Big restructuring of nixpkgs
+  - Introduction of new interfaces or functions
+  - A controversial change
+
+Some changes do not require an RFC:
+
+  - Adding, updating and removing packages in nixpkgs
+  - Additions only likely to be _noticed by_ other developers-of-nix,
+    invisible to users-of-nix.
+
+If you submit a pull request to implement a new feature without going
+through the RFC process, it may be closed with a polite request to
+submit an RFC first.
+
+## What the process is
+
+In short, to get a major feature added to the Nix ecosystem, one should first
+go throught the RFC process in order to improve the likelyhood of inclusion.
+Here are roughly the steps that one would take:
+
+* Fork the RFC repo https://github.com/NixOS/rfcs
+* Copy `0000-template.md` to `rfcs/0000-my-feature.md` (where
+'my-feature' is descriptive. don't assign an RFC number yet).
+* Fill in the RFC
+* Submit a pull request. The pull request is the time to get review of
+the design from the larger community.
+* Build consensus and integrate feedback. RFCs that have broad support
+are much more likely to make progress than those that don't receive any
+comments.
+
+At this point, the person submitting the RFC should find at least one "buddy"
```

Comment:
I don't really understand the exact role of the buddy in our case. For the rust community it makes sense because of the structure of the project (the shepherd belongs to Rust's development team which is a well-defined subset of the community), but as nix community is far less structured, it seems to me that the person submitting the RFC will *de facto* be his own buddy